.github/workflows/go.yml

```diff
@@ -16,16 +16,17 @@ jobs:
 
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.21.0', 'stable' ]
```

Comment:
Why 1.21.0? Add a comment.  What about oldstable?

Reply:
1.21.0 is the one from go.mod. Sure, let's also add old stable.